backends/platform/ios7/ios7_osys_misc.mm

```diff
@@ -226,6 +226,8 @@ static inline void execute_on_main_thread_async(void (^block)(void)) {
 		inputSet->addHardwareInputSet(new JoystickHardwareInputSet(defaultJoystickButtons, defaultJoystickAxes));
 	}
 
+  inputSet->addHardwareInputSet(new KeyboardHardwareInputSet(defaultKeys, defaultModifiers));
```

Comment:
```suggestion
	inputSet->addHardwareInputSet(new KeyboardHardwareInputSet(defaultKeys, defaultModifiers));
```

It looks like you used spaces instead of tabs for indentation.
See https://wiki.scummvm.org/index.php?title=Code_Formatting_Conventions#Tab_indents.2C_with_tabstop_at_four_spaces

Reply:
@yoshisuga I've fixed the configuration of the keymapper here, waiting to be merged.
https://github.com/scummvm/scummvm/pull/5187/commits/a3eada66832b8c9f5e911ee1ad6c2c7913317a67

If you could commit the suggestion from @criezy above (you could click on the "Commit suggestion" button in github) I think this should go in.

Another minor thing is that the commit header should mention IOS7 since that's the name of the port. We previously had an iPhone port targeting iOS versions less than 7 but that has been removed.
But perhaps we can allow IOS as well. What do you say @criezy ?

Reply:
I committed the suggestion for the spacing, I'll be aware of that in the future. 

I saw the commit for the input configurations and it makes sense, thanks for all the comments and the discussion. I'm glad we can at least get the default mapping working and address the gamepad mapping as well.